.circleci/config.yml

```diff
@@ -23,16 +23,16 @@ commands:
               - slack/notify:
                   channel: mobile-bots
                   event: pass
-                  template: basic_success_1   
+                  template: basic_success_1
                   branch_pattern: main
         - slack/notify:
             channel: mobile-bots
             event: fail
-            template: basic_fail_1   
+            template: basic_fail_1
             branch_pattern: main
             mentions: '@here'
   install-gh-cli:
-      steps:        
```

Comment:
Not sure why we have so many trailing spaces in build configuration. Not too critical, but decided to remove them.